database_wrapper.py

```diff
@@ -408,14 +408,36 @@ def get_all_qa_pairs(self):
 
         return result
 
+    def get_all_unvalidated_qa_data(self, numQueries: int):
+        qa = QuestionAnswerPair
+
+        query_session = self.session.query(
+            qa.id, qa.can_we_answer, qa.question_format, qa.answer_format, qa.verified, qa.answer_type
+        ).filter(qa.verified == 0).limit(numQueries)
+        result = query_session.all()
+        unvalidated_qa_pairs = []
+        for qa_pair in result:
+            formatted_QA_pair = {
+                "id": qa_pair[0],
+                "can_we_answer": qa_pair[1],
+                "question_format": qa_pair[2],
+                "answer_format": qa_pair[3],
+                "verified": qa_pair[4],
+                "answer_type": qa_pair[5].name,
+            }
+            unvalidated_qa_pairs.append(formatted_QA_pair)
```

Comment:
We might be able to leverage the `Entity` class, replacing this section with:
```
unvalidated_qa_pairs = [qa.get_data() for qa in result]
```

Reply:
The SQL database doesn't return a `QuestionAnswerPair` list, so I don't think we can use the `get_data()`. The query returns a double list which each inner list being the data